MultiProjectRepository/rust-publish-release.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything failss
+
+SEPARATOR="
+
+"
+
+if [ ! -f "RUST_PACKAGE_VERSION.txt" ]
+then
+    echo "RUST_PACKAGE_VERSION.txt not found."
+    exit 1
+fi
+
+PACKAGE_VERSION=$(<RUST_PACKAGE_VERSION.txt)
+PACKAGE_RELEASE_NOTES=$(<RUST_PACKAGE_RELEASE_NOTES.txt)
+PACKAGE_URL="https://crates.io/crates/doublets/$PACKAGE_VERSION"
```

Comment:
You should read package name from the file too.

Reply:
> You should read package name from the file too.

How?

Reply:
https://github.com/linksplatform/Scripts/blob/main/MultiProjectRepository/read_rust_package_info.sh#L9 the name of the package is in this file.